packages/backend/src/payment-method/ilp/service.ts

```diff
@@ -113,6 +113,26 @@ async function getQuote(
         options.receiveAmount?.value || options.receiver.incomingAmount?.value
     }
 
+    // Probing quote to get the exchange rate
```

Comment:
```suggestion
    // Probing quote to get the exchange rate even if amountToSend is less than 0 because of fees
```
or something like this, just to explain why it might be 0